diff --git a/.go-version b/.go-version
index 7a429d68a3..2f4320f67f 100644
--- a/.go-version
+++ b/.go-version
@@ -1 +1 @@
-1.24.6
+1.24.4
diff --git a/go.mod b/go.mod
index 85f7d5a9b8..682261e3cd 100644
--- a/go.mod
+++ b/go.mod
@@ -9,7 +9,7 @@ module github.com/openbao/openbao
 // updated. If we choose not to do so at some point (e.g. because we don't want some new
 // semantic related to Go module handling), this comment should be updated to explain that.

-go 1.24.6
+go 1.24.4

 replace github.com/openbao/openbao/api/v2 => ./api

